pkg/apis/datavolumecontroller/v1alpha1/types.go

```diff
@@ -95,7 +95,8 @@ type DataVolumeSourceHTTP struct {
 // DataVolumeStatus provides the parameters to store the phase of the Data Volume
 type DataVolumeStatus struct {
 	//Phase is the current phase of the data volume
-	Phase DataVolumePhase `json:"phase,omitempty"`
+	Phase    DataVolumePhase    `json:"phase,omitempty"`
+	Progress DataVolumeProgress `json:"progress,omitempty"`
```

Comment:
so when looking at the status if this is set I see something like:

> Progress:  15.03

It might be nice instead to use something like:

>PercentageDownloaded: 15.03

It's sort of interesting though, I have a failure for  URL not found, but according to the status I was at 20+ before it failed and started crash loop.

Reply:
This is why I threw it up here, I am not completely happy with it yet, but having trouble determining all the nifty edge cases. I think it would make sense to have a retry counter in the status as well. Should be relatively simple to add, simply count the crash loops.

As for the name, this will be used for cloning and other types as well, where 'Download' doesn't make sense, so I went with the more generic 'progress' I make it a float, but maybe string with a % at the end would make sense?

Reply:
`PercentagePopulated: xxx` ?

Honestly not a big deal, just wasn't intuitive when I first saw it is all.

Reply:
I talked to Matt about it, and he honestly couldn't think of a better name besides progress due to the nature of the source of the data, it could be clone/import/upload/whatever. He did want me to make it a string and append % to make it clearer what it is.

Reply:
fine by me